docs/packages/omc3/workflow.md

```diff
@@ -1,7 +1,220 @@
 # OMC3 Software Workflow
 
-!!! todo
-    Description of a typical analysis workflow. With easy examples for first-timers.
+## The Analysis Workflow
 
-!!! todo
-    Also add smaller examples for individual entry-points, scripts and plotting.
+A typical workflow with `omc3` consists in performing analysis of measurement or simulation files, and eventually calculating corrections to apply.
+While `omc3` provides python modules to handle the different aspects of these tasks, it also provides entrypoint scripts to be called from the commandline.
+
+The first step in the workflow consists in a harmonic frequency analysis performed with the `harpy` module, while the second one is an optics analysis performed with the `measure_optics` module.
```

Comment:
It is not wrong but I would say that the natural step is to first create a model and then do hole_in_one. In that way you can also get the optics functions.

Reply:
That's the last part of the page, which mentions an actual `hole_in_one`. For the detailed part I'd rather separate the two.